openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -180,6 +191,10 @@ def test_get_form_passing_auth(self):
         response = self.get_with_bearer_token('/oauth2-test/')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_get_form_passing_auth_with_dot(self):
+        response = self.get_with_bearer_token('/oauth2-test/', token=self.dot_access_token.token)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
     @unittest.skipUnless(oauth2_provider, 'django-oauth2-provider not installed')
     def test_post_form_passing_auth_url_transport(self):
         """Ensure GETing form over OAuth with correct client credentials in form data succeed"""
```

Comment:
self.get_with_bearer_token uses csrf_client, so I don't think these tests are the issue....  Maybe in auth_exchange.